Cargo.toml

```diff
@@ -52,3 +54,4 @@ olpc-cjson = "0.1.1"
 rstest = "0.11"
 hmac = "0.11"
 tracing-subscriber = "0.3.7"
+testcontainers = "0.12.0"
```

Comment:
perhaps this should be under `dev-dependencies`?

https://doc.rust-lang.org/rust-by-example/testing/dev_dependencies.html

Reply:
Ah... It already is. Github didn't expand the list far enough. LGTM!